common/src/main/scala/hmda/parser/filing/lar/LarParser.scala

```diff
@@ -23,11 +23,10 @@ trait LarParser {
     }
 
   def validateDoubleFieldReturnString(value: String, parserValidationError: ParserValidationError): LarParserValidationResult[String] =
-    Try(value.toDouble) match {
-      case Success(_) => if ("[-\\d.]+".r.matches(value)) value.validNel else parserValidationError.invalidNel
+    Try(BigDecimal(value)) match {
+      case Success(i) => i.bigDecimal.toPlainString.validNel
```

Comment:
if we still want to prevent sci-note input from users, we need to have the regex portion